draft-ietf-quic-transport.md

```diff
@@ -135,20 +142,17 @@ unidirectional streams, which allow a single endpoint to send data. A
 credit-based scheme is used to limit stream creation and to bound the amount of
 data that can be sent.
 
-The QUIC handshake combines negotiation of cryptographic and transport
-parameters.  The handshake is structured to permit the exchange of application
-data as soon as possible.  This includes an option for clients to send data
-immediately (0-RTT), which might require prior communication to enable.
+Application data can be reliably delivered by QUIC. An acknowledgment-based
+algorithm for detecting and recovering from loss of data is described in
+{{QUIC-RECOVERY}}. QUIC depends on congestion control to avoid network
+congestion. An exemplary congestion control algorithm is also described in
+{{QUIC-RECOVERY}}.
```

Comment:
I feel like it's worth mentioning that QUIC provides the feedback necessary to implement such algorithms, but doesn't mandate a particular one.

Reply:
I tried, but couldn't find anything satisfactory.  I don't think that ECN needs top-billing any more than the spin bit or any of the other minor protocol features we have.

Reply:
I'm not suggesting calling out ECN specifically, but making sure the general focus of -transport versus -recovery is clear.  Transport defines feedback mechanisms, but no algorithm.  Recovery provides *an* algorithm, but not *the* algorithm.

Perhaps as simple as:

> QUIC provides the necessary feedback to implement reliable delivery and congestion control. An algorithm for detecting and recovering from loss of data is described in....

Reply:
OK, now I think that I get it.  And I like it.  Thanks.